src/fetch.js

```diff
@@ -1,13 +1,23 @@
 import axios from 'axios';
 import { isObject, forEach, set, castArray, startsWith } from 'lodash';
 
-module.exports = async (endpoint, ctx) => {
+module.exports = async (entityDefinition, ctx) => {
   const { apiURL, queryLimit, jwtToken, reporter } = ctx;
 
+  const { endpoint, api } = entityDefinition;
+
+  // Place global params first, so that they can be overriden by api.qs
+  const params = { _limit: queryLimit, ...api?.qs };
+
+  // Retrieve qs params if defined (or empty string instead)
+  const qs = Object.keys(params)
```

Comment:
this isn't necessary with axios you can pass the query string as an object into the params property of the options

example:

```js
const requestOptions = {
  method: 'GET',
  url: apiBase,
  params: { _limit: 100, ...api?.qs },
  headers: {}
}

await axios(requestOptions)
```

Reply:
Right, better like this... Fixed in commit below